app/src/main/java/org/mifos/mobile/ui/fragments/TransferProcessFragment.java

```diff
@@ -77,11 +78,15 @@
      * @param type    enum of {@link TransferType}
      * @return Instance of {@link TransferProcessFragment}
      */
-    public static TransferProcessFragment newInstance(TransferPayload payload, TransferType type) {
+    public static TransferProcessFragment newInstance(
```

Comment:
Passing two more parameter does not sound good. I would suggest adding two more fields in `TransferPayload`
1. `fromAccountNumber` and 2. `toAccountNumber`

and annotate them with ` @Transient` so it will be ignored in the retrofit call.